functions/src/services/validate-request.ts

```diff
@@ -0,0 +1,40 @@
+import { body, validationResult } from "express-validator";
+import { NextFunction, Request, Response } from "express";
+
+/**
+ *
+ * Function to enforce a required field in request body
+ *
+ * @param fieldName - name of field that needs to be enforced as required
+ * @param onErrorMsg - if this field is not included, an error object will be returned including the message stored in this variable
+ *
+ *
+ */
+export const requireField = ({ fieldName, onErrorMsg }: { fieldName: string; onErrorMsg: string }) => {
```

Comment:
Extract the `{ fieldName: string; onErrorMsg: string }` into a separate [type alias](https://www.typescriptlang.org/docs/handbook/2/objects.html#generic-object-types) (with documentation for each field) and also use it as the parameter type for `validateRequest`.